browser/src/router.ts

```diff
@@ -5,6 +5,7 @@ import { RouterMaps } from "./samples/maps/router";
 import { RouterExcel } from "./samples/excel/router";
 import { RouterLayouts } from "./samples/layouts/router";
 import { RouterEditors } from "./samples/editors/router";
+import { RouterData } from "./samples/data/router";
```

Comment:
As per Martin's request we need to rename the data forlder to something else. He suggested "inputs".